adoc/extensions/sycl_khr_includes.adoc

```diff
@@ -0,0 +1,626 @@
+- [code]#reduction# interface
+- [code]#property::reduction::initialize_to_identity# property
+- [code]#reducer# class
+- All [code]#property::reduction# types defined by <<sec:reduction-properties>>
+
+This header also includes [code]#<sycl/khr/includes/functional># for convenience
+
+[[sec:khr-includes-interop-handle]]
+=== [code]#<sycl/khr/includes/interop_handle># header
+
+This header contains:
+
+- [code]#interop_handle# class
+
+[[sec:khr-includes-kernel-bundle]]
+=== [code]#<sycl/khr/includes/kernel_bundle># header
+
+This header gives access to kernel bundles and related functionality as defined
+by <<sec:interfaces.bundles>>. It
+contains:
+
+- [code]#bundle_state# enumeration
+- [code]#kernel_id#, [code]#kernel#, [code]#device_image# and
+  [code]#kernel_bundle# classes
+- All [code]#info::kernel# information descriptors defined by
+  <<sec:kernel-info-descriptors>>
+- Free functions like [code]#get_kernel_bundle#, [code]#has_kernel_bundle#,
+  [code]#compile#, [code]#link#
+- [code]#specialization_id# class
+
+[[sec:khr-includes-exception]]
+=== [code]#<sycl/khr/includes/exception># header
+
+This header gives access to error handling APIs defined by <<error-handling>>.
+It contains:
+
+- [code]#async_handler# alias
+- [code]#exception# and [code]#exception_list# classes
+- [code]#errc# enumeration
+- [code]#make_error_code# and [code]#sycl_category# free functions
+- [code]#std::is_error_code_enum# type trait specialization for
+  [code]#sycl::errc#
+
+[[sec:khr-includes-half]]
+=== [code]#<sycl/khr/includes/half># header
+
+This header contains definition of [code]#half# class
+
+[[sec:khr-includes-vec]]
+=== [code]#<sycl/khr/includes/vec># header
+
+This header gives access to [code]#vec# class defined by <<sec:vector.type>>.
+It contains:
+
+- [code]#rounding_mode# enumeration
+- [code]#elem# struct and [code]#vec# class
+- Type aliases defined by the section 4.14.2.2 Aliases
+- [code]#__writeable_swizzle__# and [code]#__const_swizzle__# classes defined
+  by <<swizzled-vec-class>>
+
+Note that the simple swizzle functions ([code]#XYZW_SWIZZLE# and
+[code]#RGBA_SWIZZLE# defined by the table 123) are only available when the macro
+[code]#SYCL_SIMPLE_SWIZZLES# is defined before including
+[code]#<sycl/khr/includes/vec>#.
+
+[[sec:khr-includes-marray]]
+=== [code]#<sycl/khr/includes/marray># header
+
+This header gives access to math array types defined by <<sec:marray.type>>.
+It contains:
+
+- [code]#marray# class
+- Type aliases defined by the section 4.14.3.2 Aliases
+
+[[sec:khr-includes-atomic]]
+=== [code]#<sycl/khr/includes/atomic># header
+
+This header contains:
+- [code]#atomic_fence# function
+- [code]#memory_order#, [code]#memory_scope# enumerations
+- [code]#memory_order_traits# struct and [code]#atomic_ref# class as defined
+  by <<sec:atomic-references>>
+- [code]#atomic# class and free functions like [code]#atomic_store#,
+  [code]#atomic_fetch_and# as defined by the section 4.15.4 Atomic types
```

Comment:
Hard coded section number

Reply:
Replaced with a proper reference in a55dc244a36c51cdac3627474ba28cd9ead565b5